www/js/user_page.js

```diff
@@ -308,7 +310,9 @@ function initUserPage() {
       // If it's an array (either switch or food preferences)
       if (typeof value === 'object') {
         if (key == 'food_preferences') {
-          if (value.length != 0) {
+          if (value.length === 0 || value.length === 1 && value[0] === 'Annat') {
```

Comment:
When does it occur that the only value of `food_preferences` is `Annat`? When I test locally I can never get `Annat` to appear in the array. Am I missing something? If this is true and I'm not just stupid the else statement can be shortened as well.

Reply:
If `Annat` is specified in the app then before `prepareFormData` it is included in the `food_preferences` array. The additional check to see if `Annat` is the only element in the array is needed in the case that you have `Annat` and one more food preferences, and then remove this other preferences. Previously, this other preferences would not be removed from the user in this case.

Reply:
Okay, sounds good!